op-program/client/l2/engineapi/block_processor.go

```diff
@@ -152,15 +152,9 @@ func (b *BlockProcessor) Assemble() (*types.Block, types.Receipts, error) {
 	body := types.Body{
 		Transactions: b.transactions,
 	}
-	if b.dataProvider.Config().IsPrague(b.header.Number, b.header.Time) {
```

Comment:
So this is identical to the op-geth change, would it be better to simply add a `!isIsthmus` check here rather than remove the if-statement?

Reply:
Sure, I can update that. The only difference here is we don't use the requests array at all here.

Reply:
Thanks. Yeah that's alright with the requests array. It's easier to spot if something changes in a future hardfork with similar logic between op-geth and the program.